docs/src/static-pages/accessibility.mdx

```diff
@@ -4,6 +4,8 @@ subtitle: To ensure that we are building the best possible products that work fo
 category: guidelines
 ---
 
+# Table of Contents
```

Comment:
As there are multiple sections in this page with large grouped sections - we only put one `#` here so we can get the major h1 headings (We could always make the h1s to h2s, and go down a size if we prefer?)